recipe/meta.yaml

```diff
@@ -30,15 +34,27 @@ test:
     - hatch_jupyter_builder
   commands:
     - pip check
-    - pip install -e ".[test]"
-    - python -m pytest -vv
+# Note windows tests supressed because of problems with utils.run(shlex.split(cmd))
+#    - python -m pytest -k "not (test_run or test_normalize_cmd)" -vv  # [win]
+    - pip install -e ".[test]" # [not (win or s390x)]
+    - python -m pytest -vv     # [not (win or s390x)]
   requires:
+    - hatchling
     - pip
+    - pytest               # [ not win]
+    - pytest-cov           # [ not win]
```

Comment:
`pytest-cov` is typically not required in regular `pytest` command. Does it fail if you don't include it? If not, you may want to check out the other test dependencies, too.